### PR TITLE
feat : add getbooks feature

### PR DESCRIPTION
**Pull request by Noklent-Fardian**
I don't know how it works.

### CODE REVIEW

File: src/main/java/repository/BookRepository.java [hunk 1]
```diff
@@ -3,16 +3,27 @@
...
 import java.util.List;
 
+
 public class BookRepository implements IBookRepository {
+    SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+    private final List<Book> books = new ArrayList<Book>();
+    public BookRepository() throws ParseException {
+        books.add(new Book("123","Buku 1","Musashi","PT. Berkah",dateFormat.parse("2020-03-03"),200));
+        books.add(new Book("124","Buku 2","Albert","PT. Berkah",dateFormat.parse("2020-03-03"),230));
+    }
```
Comment (elianiva) [inline src/main/java/repository/BookRepository.java]: classnya ngga boleh ngethrow `ParseException`, buat dummy data cari cara lain biar ngga perlu pake `SimpleDateFormat`

**Review by elianiva — CHANGES_REQUESTED** [review]

File: src/main/java/repository/BookRepository.java (hunk 1 above)
Comment (Noklent-Fardian) [reply src/main/java/repository/BookRepository.java]: Rogers!

**Review by elianiva — APPROVED** [review]
LGTM

**Comment by Noklent-Fardian** [comment]
Aye aye ,sir. o7

**Pull request merged**